### PR TITLE
Stop data collector

### PR DESCRIPTION
**Pull request by valouvaliavlo**

### CODE REVIEW

File: octobot_backtesting/api/exchange_data_collector.py [hunk 1]
```diff
@@ -35,6 +35,8 @@ async def initialize_and_run_data_collector(data_collector):
     await data_collector.start()
     return data_collector.file_name
 
+async def stop_data_collector(data_collector):
+    return await data_collector.abort()
```
Comment (Herklos) [inline octobot_backtesting/api/exchange_data_collector.py:39]: `abort()` doesn't exist in `DataCollector` class, why not calling `datacollector.stop()` ?

Comment (valouvaliavlo) [reply octobot_backtesting/api/exchange_data_collector.py:39]: It was for my test, but right, it can be removed now

**Review by Herklos — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,23 +1,38 @@
...
+- BacktestingApi add stop_data_collector
 
 ## [1.6.24] - 2021-09-19
+
 ### Updated
+
 - bump requirements
 
 ## [1.6.23] - 2021-09-10
+
```
Comment (Herklos) [inline CHANGELOG.md:21]: Why ?

**Review by Herklos — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (valouvaliavlo) [reply CHANGELOG.md:21]: surely pycharm !

File: octobot_backtesting/api/exchange_data_collector.py [hunk 3]
```diff
@@ -36,6 +36,10 @@ async def initialize_and_run_data_collector(data_collector):
     return data_collector.file_name
 
 
+async def stop_data_collector(data_collector):
+    return await data_collector.stop(should_stop_database=False) if data_collector else False
```
Comment (GuillaumeDSM) [inline octobot_backtesting/api/exchange_data_collector.py:40]: 👍 
If we add kwargs, we should add a **kwargs in `data_collector.py # async def stop(self) -> None:`

Comment (valouvaliavlo) [reply octobot_backtesting/api/exchange_data_collector.py:40]: ah, yes !

**Review by GuillaumeDSM — APPROVED** [review]

**Review by Herklos — APPROVED** [review]
:+1:

**Pull request merged**